include/deal.II/fe/fe_values.h

```diff
@@ -3617,6 +3624,14 @@ class FEValues : public FEValuesBase<dim, spacedim>
            const Quadrature<dim> &             quadrature,
            const UpdateFlags                   update_flags);
 
+  /**
+   * TODO
+   */
+  FEValues(const Mapping<dim, spacedim> &      mapping,
+           const FiniteElement<dim, spacedim> &fe,
+           const hp::QCollection<dim> &        quadrature,
+           const UpdateFlags                   update_flags);
```

Comment:
This is a new constructor for non-hp `FEValues` taking a `hp::QCollection` containing a quadrature rule for each face.